runtime/core/websocket/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_library(websocket STATIC
+  websocket_client.cc
+  websocket_server.cc
+)
+target_link_libraries(websocket PUBLIC decoder)
```

Comment:
Is it necessary to build the server and client together? Because the clinet does not need the decoder library.

Reply:
It's just for unified cmake for every module.

Reply:
Just like what  we did  for `grpc` before.